fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -125,28 +125,17 @@
     private static final Sort sort = new Sort();
 
     /*
-     * This is our in-memory cache of SearchParameter objects. The cache is
-     * organized at the top level by tenant-id,
-     * with the built-in (FHIR spec-defined) SearchParameters stored under the
-     * "built-in" pseudo-tenant-id.
-     * SearchParameters contained in the default tenant's
-     * extension-search-parameters.xml file are stored under the
-     * "default" tenant-id, and other tenants' SearchParameters (defined in their
-     * tenant-specific
-     * extension-search-parameters.xml files) will be stored under their respective
-     * tenant-ids as well. The objects
-     * stored in our cache are of type CachedObjectHolder, with each one containing
-     * a Map<String, Map<String,
-     * SearchParameter>>. This map is keyed by resource type (simple name, e.g.
-     * "Patient"). Each object stored in this
-     * map contains the SearchParameters for that resource type, keyed by
-     * SearchParameter name (e.g. "_lastUpdated").
-     * When getSearchParameter(resourceType, name) is called, we'll need to first
-     * search in the current tenant's map,
-     * then if not found, look in the "built-in" tenant's map. Also, when
-     * getSearchParameters(resourceType) is called,
-     * we'll need to return a List that contains SearchParameters from the current
-     * tenant's map (if present) plus those
+     * This is our in-memory cache of SearchParameter objects. The cache is organized at the top level by tenant-id,
+     * with the built-in (FHIR spec-defined) SearchParameters stored under the "built-in" pseudo-tenant-id.
+     * SearchParameters contained in the default tenant's extension-search-parameters.xml file are stored under the
```

Comment:
want to use extension-search-parameters.json instead of extension-search-parameters.xml ?

Reply:
```suggestion
     * SearchParameters contained in the default tenant's extension-search-parameters.json file are stored under the
```